04_environment/index.tex

```diff
@@ -1 +1,400 @@
-\chapter{Game Environment}
\ No newline at end of file
+\chapter{Environment}
+\section{Foraging}
+\subsection{Foraging Background}
+
+Foraging is a base requirement that is needed as it is a method of obtaining more resources from the starting amount.Foraging is based on the concept of inputting resources to get a return.\\
+
+\subsubsection{Underlying concept}
+The goal of foraging in this “game” is to introduce a certain dilemma to the agents to further understand the behaviours of multiple (6) intelligent agents working together to reach a goal with different personal goals. By having 2 methods of foraging it introduces the concept of decision making between the islands allowing for strategic interactions to maximise personal benefits whilst still reaching the overarching goal of survival.\\  
+
+The inspiration for our concept of foraging comes from the stag hunt game furthermore it introduces elements from it. Specifically it brings in the concept of \textbf{no dominant strategy} and how \textbf{ one agents choice can impact other agents.}\\
+
+The concept of stag hunt is that each player selects a method of hunting and if they differ then the social welfare (total payoff) is significantly less than if they both go stag hunting. The selection of stag means the agent is going for higher \textbf{payoff}whilst the selection of hare indicates that the agent is going for \textbf{risk aversion} as both agents are required to stag hunt to get any personal welfare. \\
```

Comment:
space after \textbf{payoff} !!

Reply:
use of personal pronouns such as our

Reply:
line 12 is made up of two quite long sentences. Consider re-writing and revising grammar for clarity because the point is strong but it gets lost in the sentence a bit